src/tests/CullingTest.cpp

```diff
@@ -26,7 +28,22 @@ namespace Mn = Magnum;
 using esp::assets::ResourceManager;
 using esp::scene::SceneManager;
 
-TEST(CullingTest, computeAbsoluteAABB) {
+namespace Test {
```

Comment:
```suggestion
namespace Test { namespace {
```

Not strictly needed, but on GCC and Clang this causes useful warnings to be printed when you have accidentally unused variables or functions in the test, which I think is very useful.